movement/filtering.py

```diff
@@ -109,13 +114,17 @@ def interpolate_over_time(
     at the first data point after a gap and the last value before a gap.
 
     """
-    data_interpolated = data.interpolate_na(
-        dim="time",
-        method=method,
-        use_coordinate=False,
-        max_gap=max_gap + 1 if max_gap is not None else None,
-        **kwargs,
-    )
+    if method in ["bfill", "ffill"]:
+        data_interpolated = getattr(data, method)(dim="time", **kwargs)
```

Comment:
As noted by the code coverage report, this new lines need to be properly tested. Otherwise we cannot be sure that the added functionality behaves as we expect.

Reply:
It would be useful to also implement a case that catches if the method argument is not one of the supported values.